requirements.txt

```diff
@@ -1,3 +1,4 @@
+vmprof==0.4.12
```

Comment:
Right that is missing. I need to pick up on that when I fight PyStone again with new C type optimization.